### ISSUE TITLE
Getting Bad response from server on first startup

### ISSUE THREAD

**Issue by ballonnetje** [Bercik1337/rt-auto-install#51]
Hello, trying to install on a new server.. had a working install on a different server and worked great. Now I'm getting this error and I can't seem to get rid of it.. 

```
[15.08.2023 17:23:25] Bad response from server: (200 [parsererror,getuisettings]) <?php require_once( 'WebUISettings.php' ); $settings = WebUISettings::load(); $json = $settings->get(); CachedEcho::send($json,"application/json",false);
[15.08.2023 17:23:25] JS error: [http://192.168.178.189:8999/rutorrent/ : 1] SyntaxError: expected expression, got '<'
```

I'm trying to install on a server with some other stuff already installed: Gitlab server (port 80 etc.), some cctv security stuff, plex, and now trying to install rtorrent.
Also having trouble on launching the rtorrent service. From time to time I'm able to fix it, but now after  a couple reconfigurations using different configs and multiple re-installs using your installer - things are kinda stuck. Not sure why it's not working. 

I got the above error in the very beginning (still getting it) when the service was running fine. 

Getting this when trying to Start service:
```
The unit rtorrent.service has entered the 'failed' state with result 'exit-code'.
Aug 15 15:35:13 server systemd[1]: Failed to start rtorrent (in tmux).
```

Basically this probably means there is already an instance running. So killing pid with -9. Sometimes working, sometimes its not.

---

Okay during typing all this I rebooted and not at least the rtorrent service is up. Still get the bad response though. 
```

$ xmlrpc localhost/RPC2 system.client_version
Failed.  Call failed.  HTTP response code is 301, not 200.  (XML-RPC fault code -504)

```

**Comment by ballonnetje** [comment]
apparantly the php module in apache2 wasn't enabled. The installation did handle this earlier right? Very strange.

**Comment by Bercik1337** [comment]
which distro? 
any messages during installation?

**Comment by ballonnetje** [comment]
Ubuntu (server) 22.04.3 LTS  

No messages during installation.

**Comment by Bercik1337** [comment]
@ballonnetje sorry, i can't repro
just installed fresh Ubuntu Desktop and Ubuntu Server 22.04, in both there were no problems.
I managed to download Debian netinstall iso without any problem using GUI (rutorrent).


I see you are trying to connect on 8999 port:
`[15.08.2023 17:23:25] JS error: [http://192.168.178.189:8999/rutorrent/ : 1] SyntaxError: expected expression, got '<'`

Where did this 8999 come from?

**Comment by ballonnetje** [comment]
port 8999 is just something i tried, since port 80 is taken by the gitlab nginx.

**Comment by ballonnetje** [comment]
> @ballonnetje sorry, i can't repro just installed fresh Ubuntu Desktop and Ubuntu Server 22.04, in both there were no problems. I managed to download Debian netinstall iso without any problem using GUI (rutorrent).
> 
> I see you are trying to connect on 8999 port: `[15.08.2023 17:23:25] JS error: [http://192.168.178.189:8999/rutorrent/ : 1] SyntaxError: expected expression, got '<'`
> 
> Where did this 8999 come from?

Also.. this isn't a clean server install.. first installed gitlab and some other stuff.

**Comment by ballonnetje** [comment]
@Bercik1337 Thanks for your response. Still trying to fix this though. 

Uninstalled, removed, purged, tried qbittorrent (worked but f-ed up on ssl certs and stuff so trying this again)

During install, it failed on installing scgi. Had to remove that first. After that it did succeed at installation but got the same 200 error. Had to remove php module and reinstall and re-enable. 200 error gone, same JS error. 

Moved to scgi port 5005, editted rutorrent conf, editted apache2 conf. Both are running without errors (as far as systemctl is concerned). 

It looks like it's not automatically installing required modules maybe? Could this be a problem? Maybe any dependencies I'm missing?

**Comment by Bercik1337** [comment]
@ballonnetje i tried using 8999 myself last night and it works great.

Now we have answer, if you have something else running, or it's not "clean" install I cant guarantee anything. There is just so many possibilities people can mess around the systems and i simply don't have the time to go through all exceptions and try to code them. Perhaps try other way around, fresh OS install -> my script -> gitlab then. Maybe that will help - if not, you need to make decision, or setup 2 VMs one for each app.

**Comment by ballonnetje** [comment]
OMG... Just had to clear my browser cache... it's all working now!

**Comment by Bercik1337** [comment]
glad to hear that :) happy seeding
